Sources/FileSystem/Model/BetaGroup.swift

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+
+public struct BetaGroup: Codable, Equatable {
+
+    public typealias EmailAddress = String
+
+    public let id: String?
```

Comment:
Generally `struct` properties should be `var` as the `struct` being declared constant with `let` will cascade to the properties.